packages/react-day-picker/src/contexts/DayPicker/DayPickerProvider.tsx

```diff
@@ -54,6 +54,7 @@ export function DayPickerProvider(props: DayPickerProviderProps): JSX.Element {
   if (toDate) {
     modifiers.disabled.push({ after: toDate });
   }
+  modifiers.today = [today];
```

Comment:
Following the advice of `// TODO: modifiers from context should not be overridden` in `useModifiers`, I am properly assigning the modifiers to `[today]` here (the modifiers from the context), so that we are not overriding them in `useModifiers`